psrp/script.yaml

```
@@ -2691,8 +2691,8 @@
     Si vous le voyez, pourriez-vous le saluer de ma part?<wait>
   pt-br: |
     Eu tenho um amigo em Bartevo.<wait more>
-    Por causa de todas aquelas poças de lava, acho que as coisas estão bem ruins por lá.<wait more>
-    Se caso o vir, poderia cumprimentá-lo por mim?<wait>
+    Por causa de todas aquelas poças de lava, acho que as coisas estão bem feias por lá.<wait more>
+    Se caso o vir, poderia dizer ‘oi’ para ele por mim?<wait>
```

This can be shortend a bit
```suggestion
    Se caso o vir, poderia dizer ‘oi’ por mim?<wait>
```

---

> This can be shortend a bit

sounds nice, thx